main/java/TestForVirtualItemClass.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Disabled
+public class TestForVirtualItemClass extends BeforeAfter {
+
+    @Test
+    void scenarioSetSizeOnDisk() {
```

Comment:
RealItem and VirtualItem classes were designed as simple classes without any business logic. There is no needs to write tests for these classes at all, the only method for which it can be done - toString(). Do not write tests for simple operations (get/set). Please remove test and add for toString() (not required)

Reply:
Correction is applied.